src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav__link">
+        <ul class="nav">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
```
entire navigation is definitely not a `nav__link`